packages/ts/file-router/src/runtime/RouterConfigurationBuilder.ts

```diff
@@ -150,16 +152,7 @@ export class RouterConfigurationBuilder {
    */
   withLayout(layoutComponent: ComponentType): this {
     function applyLayouts(routes: readonly RouteObject[]): readonly RouteObject[] {
-      const nestedRoutes = routes.map((route) => {
-        if (route.children === undefined) {
-          return route;
-        }
-
-        return {
-          ...route,
-          children: applyLayouts(route.children),
-        } as RouteObject;
-      });
+      const nestedRoutes = routes.map((route) => route);
```

Comment:
The map call seems to do nothing. Could we remove it then?

Reply:
I think it was left just because routes was readonly and it "broke" something.

Reply:
Yes so I get a `TS4104: The type readonly RouteObject[] is readonly and cannot be assigned to the mutable type RouteObject[]` when trying to use routes directly.